app/controllers/sessions_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
   end
 
   def create
-    user = User.find_by_email(params[:email])
+    user = User.find_by_email(params[:email].downcase)
```

Comment:
@case-eee This seems kind of code-smelly. Have you seen a better way to be case insensitive?